End of 2021 (#602)
ClaudeMetz/Alt-F4: submissions/end-2021/end-2021.md

# Looking Back To Factorio's Early Days<author>Many People</author>

## Old school Factorio [<author>JD-Plays</author>](https://www.youtube.com/jdplays)

> Heya I'm JD, a YouTuber & Twitch streamer whos clocked up a few thousand hours in Factorio, somewhere over the 10K hour mark at this point, but it wasn't always this way, I got into Factorio in version 13ish, having tried out the earlier versions and deciding the game wasn't for me.

Reviewer: ```suggestion
Heya, I'm JD! I'm a YouTuber & Twitch streamer who's clocked up a few thousand hours in Factorio, somewhere over the 10K hour mark at this point, but it wasn't always this way. I got into Factorio in version 13ish, having tried out the earlier versions, and deciding the game wasn't for me.
```

> So lets talk about the old versions of Factorio. I think we can break Factorio timeline into several eras or generations of Factorio. First we have modern Factorio, Which would be version 0.15+ This is where Military, Production, High-tech and space science was first added to the game, and base became measured in Science Per Minute rather than Rockets Per Minute which was the previous standard. Version 0.15 to 1.1 hasn't had a lot changes. sure there has been small tweaks to recipes and combat, and a grafifical update or 2, but if you gave someone who has never seen Factorio prior to 1.1 a old copy of version 0.15 their main questions would revolve around why there is no copy and paste!

Reviewer: ```suggestion
So let's talk about the old versions of Factorio. I think we can break the Factorio timeline up into several eras or generations of Factorio. First we have modern Factorio, which I consider to be version 0.15 and later. This is the version where Military, Production, Utility and Space science packs were first added to the game, and where bases became measured in Science Per Minute, rather than Rockets Per Minute, which was the previous standard. Version 0.15 to 1.1 hasn't had a lot of changes. Sure, there have been small tweaks to recipes and combat, and a graphical update or two, but if you gave someone who has never seen Factorio prior to 1.1 an old copy of version 0.15 their main question would be why there is no copy and paste!
```

> Factorio 0.12 - 0.14 This is commonly referred to as old school factorio, It sort of feels like factorio a hour before dawn. You sort of know what's happening, It also looks semi familiar but something is off. Firstly the pipette tool, You know the Q button.... well that doesn't exist yet! Boilers are 1x1 tiles, and the ratio for power is 1 Pump to 14 boilers to 20 steam engines, Science ends in Purple Science, but its called Alien Science packs and it requires Alien Artefacts to create which you only obtain from killing spawners! These Artefacts are also required for tier 3 modules, along with power armour. 

Reviewer: ```suggestion
Factorio 0.12 - 0.14
This is commonly referred to as old-school Factorio. It sort of feels like Factorio wasn't fully baked yet. You kind of know what's happening, it also looks semi-familiar, but something is off. Firstly, the pipette tool. You know, the Q button... Well, that doesn't exist yet! Boilers are 1x1 tiles, and the ratio for power is 1 pump to 14 boilers to 20 steam engines. Science ends in purple science, but is called an alien science pack, and requires alien artifacts to create, which you can only obtain by killing spawners! These artifacts are also required for tier 3 modules, along with power armor. 
```

> Don't get me wrong its still factorio, its still feels like Factorio, you know a hour before dawn when your half delirious, but the core game mechanics are there. Everything functions the same way as it does now, well except for blueprints, you had to research them, then craft them, oh and they cost one red circuit to clear the blueprint! Oh and did I mention to share blueprints back then we had to share them as a screen shot! yep Blueprint strings were not a thing yet!

Reviewer: ```suggestion
Don't get me wrong, it still feels like Factorio with its strong core game mechanics, even if it's missing some polish. Everything functions the same way as it does now. Well, except for blueprints, because you had to research them, then craft them, and it cost one red circuit to clear the blueprint! Oh, and did I mention that to share blueprints back then, we had to share them as a screenshot? Yep, blueprint strings were not a thing yet!
```

> This is also when Multiplayer was first added with version 0.11, It was far from a pleasant experience, there was no latency hiding that there is now. This is around when the player base expanded first with 0.11 and Multiplayer and again in version 0.12 when latency hiding was added, and now you could play with anyone, anywhere in the world, well as long as you stayed out of combat!

Reviewer: ```suggestion
Version 0.11 is also when multiplayer was first added. It was far from a pleasant experience, as there was no latency hiding like there is now. This is around the time when the playerbase expanded: first with 0.11 which brought multiplayer, and again in version 0.12 when latency hiding was added. Now you could play with anyone, anywhere in the world. Well, as long as you stayed out of combat!
```

![JD calls for Santa in 2018](https://clips.twitch.tv/FaithfulPiercingStarLeeroyJenkins)

> So Version 0.12 Of factorio Is when I came back to Factorio. I tried it in version 0.10 which we will get to below, but in version 12 a Lot of things were added and refined, and the game was starting to feel like it does today. Bases were measured in Rockets per Minute, and your biggest bases launched about 1 rocket per minute, Mods were a thing, and Factorio already had quite a large mod repository, doing everything we see now from adding QOL features to enhancing the end game or even giving us harder biters to deal with!

Reviewer: ```suggestion
Version 0.12 is when I came back to Factorio. I tried it in version 0.10, which we will get to below, but in 0.12 a lot of things were added and refined, and the game was starting to feel more like it does today. Bases were measured in Rockets per Minute, and your biggest bases launched about one rocket per minute. Mods were a thing, and Factorio already had quite a large mod repository, doing everything we see now from adding QOL features to enhancing the end game or even giving us harder biters to deal with!
```

Factorio 0.7 - 0.11

> These special set of players tend to be one of the early Indiegogo backers, Yep im sure you know that Factorio started a kickstarter, on the other kickstarter site Indiegogo, It was started by a pair of unknown devs with a dream and they brought us the game we have today. This is actually when i started playing with version 0.10, and honestly I wasn't a fan. The game seems a bit clunky, the interface had a lot to build upon and it felt like it was a early beta, which I guess it was. If you want a good idea of what the very early versions of factorio was like check out the [2013 Trailer](https://www.youtube.com/watch?v=V1qOCAM9Syw). Back then End game was a rocket defence rather than a Rocket Silo, the rocket defence was a large building that had a place holder graphics and replaced before it was ever skinned, It had to be built, and then protect from biters for 10min to allow it time to power up, once powered it  would protect the rescue fleet who would rescue you from this biter infested planet. 

Reviewer: ```suggestion
I'm sure you know that Factorio started as a kickstarter, on the other kickstarter site Indiegogo. It was started by a pair of unknown devs with a dream, and they brought us the game we have today. This is actually when I started playing with version 0.10, and honestly I wasn't a fan. The game seemed a bit clunky, the interface left a lot to be desired, and it felt like it was an early beta, which I guess it was. If you want a good idea of what the very early versions of Factorio were like you should check out the [2013 trailer](https://www.youtube.com/watch?v=V1qOCAM9Syw). Back then, the endgame was a rocket defense, rather than a rocket silo. The rocket defense was a large building that had placeholder graphics and was replaced before it was ever given a texture. It had to be built and be protected from biters for ten minutes, after which the rescue fleet would evacuate you from this biter-infested planet.
```

> Biter were slowly evolved from 'creepers' into biters, yep they use to look like zombies from minecraft and the player looked like a power ranger! Biter also use to scream at the player when coming in to attack your structures, it was quite a horrifying experience.

Reviewer: ```suggestion
Biters were slowly evolved from 'creepers'. Yep, they used to look like the zombies from Minecraft, and the player looked like a Power Ranger! Biters also used to scream at the player when coming in to attack your structures, which was quite a horrifying experience.
```

> Oh and the car, The car is one thing that players remember as it has had a few make overs during the many years of Factorio first it looked like something ripped right out of Who Framed Rodger Rabbit, then was made more futuristic, then updated to the more steam punk buggy we have now

Reviewer: ```suggestion
Oh, and the car! The car is one thing that players remember, as it has had a few makeovers during the many years of Factorio. At first it looked like something ripped right out of *Who Framed Roger Rabbit*, then it was made more futuristic, and then it was updated to the more steampunk buggy we have now.
```

> Though the many years of Factorio not only has the game changed, but also how the players play the game has changed. Pre version 0.11 The idea of a Bus was unheard of, most people just spaghettied everything together the best they could, and the bus concept I think really originated on multiplayer servers, as it was the easists for other players to understand what a base was doing and how they could help. The same goes for diect insertion, again this was something that just wasnt done in the early days of Factorio. Copper wire went onto a belt before it was turned into green circuts, its just how things were done. I remember hearing of a fellow content creator 'FishSandwich' who made the first commonly shared red circut build with a copper cable machine feeding 8 red circuit machines, yes back then it red circuits took 20% longer to craft! so the builds were all 20% larger, can you imagine?

Reviewer: ```suggestion
Throughout the years, not only has the game itself changed, but also the way the players play it. Before version 0.11 the idea of a bus was unheard of, and most people just spaghettied everything together the best they could. I think the bus concept really originated on multiplayer servers, as it made it the easiest for other players to understand what a base was doing and how they could help. The same goes for direct insertion, as again this was something that just wasn't done in the early days of Factorio. Copper wire went onto a belt before it was turned into green circuits; it's just how things were done. I remember hearing of a fellow content creator 'FishSandwich' who made the first commonly shared red circuit build with a copper cable machine feeding 8 red circuit machines. Yes, back then red circuits took 20% longer to craft, so the builds were all 20% larger, can you imagine?
```

> Smelters were another big change, Back in the early days of factorio people designed and refined around upgrading their smelter blocks to feed blue belt, and be compatible with eclectic furnaces once they unlocked the technology, mainly because it was more coal efficient to run electric furnaces than it was to run steel furnaces for smelting. So the output of every furnace was placed onto a belt before siding loading the central belt for the output, as well as extra spacing between each stone furnace as the electric furnace was 3x3 tiles not 2x2 tiles

Reviewer: ```suggestion
Smelters were another big change. Back in the early days of Factorio people designed and refined around upgrading their smelter blocks to feed blue belts, and to be compatible with electric furnaces once they unlocked the technology. So, the output of every furnace was placed onto a belt that got sideloaded onto the central output belt, as well as having extra spacing between each stone furnace, as the electric furnace is 3x3 tiles, not 2x2 tiles.
![A demonstration of a smelter design that can be upgraded to use electric furnaces by using sideloading.](https://user-images.githubusercontent.com/32989873/147580977-e0352b85-8505-4e87-862e-36b93d8846f4.png)
```

> Factorio Has had a long roadmap with a lot of development time, and a very active fanbase with many diverse ideas about the game. I should know ive celebrated Factorio Xmas with Biter Santa back in 2018, I've waded my way through a deathworld to visit the edge of the Factorio Map [3 times for the challenge](https://www.youtube.com/watch?v=9RJ9TqRzOsg&list=PLnIE0W-m629ec9CInlPI2b2uTKQEYtKFB), I've pushed the game to its limits building a 1024 lane balancer then testing it with 1024 satellites and [launching 1024 rockets at once](https://clips.twitch.tv/GracefulBlightedBeefTooSpicy), and currently im testing my patience and the communities building a Factorio bus, at a [45 degree angle](https://www.youtube.com/playlist?list=PLnIE0W-m629fdeu7i2wZk9W0blkwbb1Ur).

Reviewer: ```suggestion
Factorio has had a long roadmap with a lot of development time, and a very active fanbase with many diverse ideas about the game. I should know, as I've celebrated Factorio Xmas with Biter Santa back in 2018; I've waded my way through a deathworld to visit the edge of the Factorio map [3 times for the challenge](https://www.youtube.com/watch?v=9RJ9TqRzOsg&list=PLnIE0W-m629ec9CInlPI2b2uTKQEYtKFB); I've pushed the game to its limits building a 1024 lane balancer, testing it with 1024 satellites and [launching 1024 rockets at once](https://clips.twitch.tv/GracefulBlightedBeefTooSpicy); and currently I'm testing my patience and the patience of the community by building a Factorio bus that's at a [45 degree angle](https://www.youtube.com/playlist?list=PLnIE0W-m629fdeu7i2wZk9W0blkwbb1Ur).
```

> Factorio has changed over the years, the players have changed over the years and im sure 2022 will bring new changes new devolpments and maybe even the expansion with even more changes to this game we all love

Reviewer: ```suggestion
Factorio has changed over the years, the players have changed over the years, and I'm sure 2022 will bring new changes, new developments, and maybe even the expansion with even more changes to this game we all love.
```

## So Rocks were absolutely awful [<author>AntiElitz</author>](https://www.twitch.tv/antielitz/about)

> I could bring up that Factorio still has not the greatest graphics of all time today, but if you look at 0.11 it has come a very far way. It's really unbearable to look at older versions as of today when I stumble over old videos on my account, but I think I going to come from a speedrunner perspective here. The most remarkable changes for me are probably all the quality of life changes that made in into the game since I started Speedrunning in 0.12.

Reviewer: ```suggestion
I could bring up that Factorio still doesn't have the greatest graphics of all time today, but if you look at 0.11 it has come a very long way. It's really unbearable to look at older versions today when I stumble over old videos on my account, but I think I'm coming from a speedrunner's perspective here. The most remarkable changes for me are probably all the quality of life changes that made it into the game since I started speedrunning in 0.12.
```

> I still remember my first base had a very weird requirement. I needed Iron and copper absolutely opposite of each other to build my base in the middle of them. Why was that so? Well it happened to be the time where belts still became decompressed on every single turn you made with them. So to save time and resources you wanted to avoid any turn possible on full lanes of plates.

Reviewer: ```suggestion
I still remember my first base had a very weird requirement. I needed iron and copper absolutely opposite of each other to build my base in the middle of them. Why was that so? Well, it happened to be the time where belts still became decompressed on every single turn you made with them. So, to save time and resources you wanted to avoid any turn possible on full lanes of plates.
```

> These days rocks are very critical for speed runner to give you a head start. However in 0.12 you could not mine rocks at all. So getting rid of any in your was needed about 10 ammunition to kill it. *So Rocks were absolutely awful.*

Reviewer: ```suggestion
These days rocks are very critical for speedruns to give you a head start. However, in 0.12 you could not mine rocks at all. Getting rid of any in your way needed about ten ammunition, *so rocks were absolutely awful*.
```

> One of my favourite mechanics pre 0.15 was that you could void any fluid in steam engines. So when setting up oil. you didn't need to crack the other liquids - you simply voided them in steam engines and had a very easy setup. There was also the possibility to place offshore pumps for water at the edge of the map. So we speedrunners did limit the map on purpose to generate a straight line for a powerplant at a desired position. Also not all seeds were different at the old times, instead you had one big master map and the seed just determinised your starting position on it. So we speedrunners kind of generated the master map and chose the seed accordingly, so you start at a desired location.

Reviewer: ```suggestion
One of my favourite mechanics pre-0.15 was that you could void any fluid in steam engines. So, when setting up oil you didn't need to crack the other liquids - you simply voided them in steam engines and had a very easy setup. There was also the possibility of placing offshore pumps for water at the edge of the map, which we speedrunners used by limiting the map's size on purpose to generate a straight line for a powerplant at a desired location. Also, not all seeds were different back in the day. Instead, you had one big master map, and the seed just determined your starting position on it. We speedrunners kind of generated the master map and chose the seed accordingly, so we could start at a desired location.
```

> When I ran 100% back in the days it required farming thousands of [alien goop](https://alt-f4.blog/ALTF4-6/#alien-artifacts-a-primitive-technology). It was really annoying. I literally went alien hunting for over 2 hours that days just annihilating the biters without any skill requiered. Really good they removed this

Reviewer: ```suggestion
When I ran the 100% category back in the day, it required farming thousands of [alien goop](https://alt-f4.blog/ALTF4-6/#alien-artifacts-a-primitive-technology). It was really annoying. I literally went alien hunting for over two hours that day just annihilating the biters without any skill required. It's really good they removed this!
```

![AntiElitz first Any% speedrun upload to YouTube in 2015. It was the world record at 3:45:58](https://youtu.be/cCiG-7gLTcg)

## The Highlights of Factorio history <author>[MojoD](https://www.twitch.tv/mojod)</author>

> Since 0.12 two releases major releases really stand out above all others, 0.13 and 0.15.  These two releases had a lot to look forward to and they delivered the foundation of what we know of the game today.

Reviewer: ```suggestion
Since 0.12 two major releases really stand out above all others: 0.13 and 0.15. These two releases had a lot to look forward to and they delivered the foundation of the game today.
```

> 0.13 was the a big step up in quality of life and addressed many of the frustrating issues with trains and combat (who remembers having to constantly cycle the 8 rotations of curved rail), removal of smart inserter and smart chest and removing a lot of annoying limitations of the circuit network as well and most significant of all buildings taking damage no longer had an alert sound.  There was also hope of seeing the Spidertron in game we were teased with months prior.  As addictive as the game was before 0.13 the game was pretty unbearable at times.

Reviewer: ```suggestion
0.13 was a big step up in quality of life and addressed many of the frustrating issues with trains and combat (Who remembers having to constantly cycle the eight rotations of curved rail?), removed smart inserters and smart chests, and removed a lot of annoying limitations of the circuit network as well. Most significant of all, buildings taking damage no longer had an alert sound. There was also hope of seeing the spidertron in the game, which we had been teased with the prior months. As addictive as the game was before 0.13, the game was pretty unbearable at times.
```

> Of all releases 0.15 I feel was the most anticipated and significant update as it completely transformed the game and was the foundation of what became the game is now.  Among the other changes the upgraded tech tree finally giving purpose to end game with infinite science.  Unfortunately releases at this stage never came out on time and would finally be released after months of delays.

Reviewer: ```suggestion
Of all releases, 0.15 I feel was the most anticipated and significant update as it completely transformed the game and was the foundation for what it is now. Among the other changes, the upgraded tech tree finally gave purpose to the endgame with infinite science. Unfortunately, releases at this stage rarely came out on time and would often be released after months of delays.
```

> The release also stands out because by the power of some creative modding, a Colonelwill and the moral support from a copper patch that bore remarkable resemblance to a duck a it was possible to build 1K Spm factory producing 0.15 infinite science in 0.14.  It also shows how far above and beyond Wube has gone over the years because not only did the 0.14 factory work in 0.15, the save and the mod also can be brought forward to the latest 1.1 release and still work as it should. 

Reviewer: ```suggestion
The release also stands out, because with the power of some creative modding, a ColonelWill, and the moral support from a copper patch that bore remarkable resemblance to a duck, it was possible to build a 1K SPM factory producing 0.15's space science packs in 0.14. It also shows how far above and beyond Wube has gone over the years, because not only did the 0.14 factory work in 0.15, but the save and the mod can also be updated to the latest 1.1 release and they'll still work as they should. 
```

## Community Thoughts and Memories

> I enjoyed all the time that I have spent in front of Factorio in an attempt to make my factory better, not realizing the time, but doing something that brought me joy. - oof2win2, Alt-F4 Contributor/Translator

> > Factorio have succeded into one thing: each time you are converting someone else to the cult, they are finding their own way to play, a way you did not thought yet. - Firerazer, Alt-F4, Chief Translator

Reviewer: ```suggestion
> Factorio has succeeded in one thing: each time you are converting someone else to the cult, they are finding their own way to play, a way you did not think of yet. - Firerazer, Alt-F4, Chief Translator
```

> > Sometimes I like to throw everything I learned about the game out of the window, install a big overhaul mod and just spend a few dozen of hours tangling up a spaghetti base. - Trasdegi, Alt-F4 Translator

Reviewer: ```suggestion
> Sometimes I like to throw everything I learned about the game out of the window, install a big overhaul mod and just spend a few dozen hours tangling up a spaghetti base. - Trasdegi, Alt-F4 Translator
```

> > My first base was spaghetti hell and a friend recommend me to use trains for better organisation. Next time he joined me, he learned that it is possible to build train spaghetti. - timestultus, Alt-F4 Reader

Reviewer: ```suggestion
> My first base was spaghetti hell and a friend recommended me to use trains for better organisation. Next time he joined me, he learned that it is possible to build train spaghetti. - timestultus, Alt-F4 Reader
```

> I enjoy seeing my bases progress from horribly inefficient to badly inefficient. There is always the possibility to make something better, it´s never perfect. - pocarski, Alt-F4 Contributor/Translator

> > The first time I played I had had this weird idea to loop back my resources back to the start to not waste them. It did not work well. - stringweasel, Alt-F4 Editor

Reviewer: ```suggestion
> The first time I played I had had this weird idea to loop back my resources to the start to not waste them. It did not work well. - stringweasel, Alt-F4 Editor
```

> I tried out the demo and ended up playing for an entire day. I spent ages mining out the entire tiny demo map and working out how to cross over belts without undergrounds using inserters. That night, I slept terribly, and dreamed of nothing but factories. I bought the full game first thing the next morning. - Bentham, [Content Creator](https://alt-f4.blog/ALTF4-36/)

> > New updates with extra content are what I miss the most, rocket silo with infinite science, nuclear update, artillery and many small changes was awesome during beta. - Trupen, [Content Creator](https://www.youtube.com/c/Trupen)

Reviewer: ```suggestion
> New updates with extra content are what I miss the most, rocket silo with infinite science, nuclear update, artillery and many small changes were awesome during the beta. - Trupen, [Content Creator](https://www.youtube.com/c/Trupen)
```

> > One of my most memorable and funny moments is from one of the Earth Map pkaythroughs. Colonel Will was "gardening" with the Orbital Ion Cannon Mod and decided it to "trim down" some trees near me. When he warned me with about 3 seconds before impact, I panicked ran straight into the blast! - Xterminator, [Content Creator](https://www.youtube.com/channel/UC5StrkKVnU2xkjV0mVJ9yTw)

Reviewer: ```suggestion
> One of my most memorable and funny moments is from one of the Earth Map playthroughs. ColonelWill was "gardening" with the Orbital Ion Cannon Mod and decided to "trim down" some trees near me. When he warned me with about 3 seconds before impact, I panicked and ran straight into the blast! - Xterminator, [Content Creator](https://www.youtube.com/channel/UC5StrkKVnU2xkjV0mVJ9yTw)
```

> idk, first time I did oil in 0.11 was pretty magical and I did the thing with logistic bots where they craft themselves. - Klonan, Wube Developer

> ![We end of by showing you Klonan's first base!](media/klonan-base.png)

Reviewer: ```suggestion
![We end off by showing you Klonan's first base!](media/klonan-base.png)
```

> > There lies a hidden gem within Factorio we have not yet fully explored.  I will never forget how much I enjoyed the multiplayer Production Battle PvP scenario that was played years ago.  Points are awarded for anything you make and removed for anything you consume.  My all time favorite moment of Factorio was that moment that our team's score went from positive 60k to negative 60k points per minute and everyone who was watching said, "What the heck is going on?!" - Rain, [Content Creator and Speedrunner](https://www.twitch.tv/rain9441)

Reviewer: ```suggestion
> There lies a hidden gem within Factorio we have not yet fully explored. I will never forget how much I enjoyed the multiplayer Production Battle PvP scenario that was played years ago. Points are awarded for anything you make and removed for anything you consume. My all-time favorite moment of Factorio was where our team's score went from positive 60k to negative 60k points per minute and everyone who was watching said, "What the heck is going on?!" - Rain, [Content Creator and Speedrunner](https://www.twitch.tv/rain9441)
```

> "The game design is so complex, they must have some crazy spreadsheets and tools to balance all this out" was also something I thought before working on the game. Then I started working and realized it was all balanced and designed by just playing the game. - Twinsen, Wube Developer

> > I remember that I almost didn't pick up Factorio because of the basic graphics, but the trailer was amazing at showing the ever growing scale that I've come to love about the game. What I really enjoyed in the pre-release era was the constant evolution of the game; new core designs were invented regularly (like Main Bus and City Blocks), weekly blogs and regular new features added (Nuclear and Infinite Science), and big overhaul mods reaching maturity (Bobs, Angels and AAI). I hope the communication from Wube starts soon for the expansion, so we can reignite the magic of being part of the evolution of game we love" - Nilaus, [Content Creator](https://www.youtube.com/c/Nilaus)

Reviewer: ```suggestion
> I remember that I almost didn't pick up Factorio because of the basic graphics, but the trailer was amazing at showing the ever-growing scale that I've come to love about the game. What I really enjoyed in the pre-release era was the constant evolution of the game; new core designs were invented regularly (like the Main Bus and City Blocks), there were weekly blogs, new features were added regularly (Nuclear and Infinite Science), and big overhaul mods were reaching maturity (Bobs, Angels and AAI). I hope the communication from Wube starts soon for the expansion, so we can reignite the magic of being part of the evolution of the game we love." - Nilaus, [Content Creator](https://www.youtube.com/c/Nilaus)
```
